src/components/CourseTable/index.jsx

```diff
@@ -71,14 +72,19 @@ class CourseTable extends React.Component {
     const {
       editors: prevEditors,
       course_run_statuses: prevCourseRunStatuses,
+      course_type: prevCourseType,
     } = qs.parse(prevProps.location.search);
     const {
       editors,
       course_run_statuses: courseRunStatuses,
+      course_type: courseType,
     } = qs.parse(this.props.location.search);
     if ((editors !== prevEditors) || (courseRunStatuses !== prevCourseRunStatuses)) {
       this.getSelectedFiltersFromUrl();
     }
+    if (courseType !== prevCourseType) {
+      this.updateFilterQueryParamsInUrl(selectedFilters);
```

Comment:
course editor and run status are coming into CourseTable component, not managed by the table component. If we manage state for type, can't it mitigate the filter reset issue? or am I missing something here?